docs/macros.md

```diff
@@ -566,23 +567,24 @@ Generates sql to build a satellite table using the provided parameters.
         ),
 
         latest_records AS (
-            SELECT c.CUSTOMER_PK, c.HASHDIFF, c.LOAD_DATE,
-                   CASE WHEN RANK()
-                   OVER (PARTITION BY c.CUSTOMER_PK
-                   ORDER BY c.LOAD_DATE DESC) = 1
-            THEN 'Y' ELSE 'N' END AS latest
-            FROM update_records as c
-            QUALIFY latest = 'Y'
+            SELECT target.CUSTOMER_PK, target.HASHDIFF, target.LOAD_DATE
+                ,RANK() OVER (
+                    PARTITION BY target.CUSTOMER_PK
+                    ORDER BY target.LOAD_DATE DESC
+                ) AS rank
+            FROM update_records AS target
+            QUALIFY rank = 1
         ),
-        
+
         records_to_insert AS (
-            SELECT DISTINCT e.CUSTOMER_PK, e.HASHDIFF, e.CUSTOMER_NAME, e.CUSTOMER_PHONE, e.CUSTOMER_DOB, e.EFFECTIVE_FROM, e.LOAD_DATE, e.SOURCE
-            FROM source_data AS e
+            SELECT DISTINCT stage.CUSTOMER_PK, stage.HASHDIFF, stage.CUSTOMER_NAME, stage.CUSTOMER_PHONE, stage.CUSTOMER_DOB, stage.EFFECTIVE_FROM, stage.LOAD_DATE, stage.SOURCE
+            FROM source_data AS stage
             LEFT JOIN latest_records
-            ON latest_records.HASHDIFF = e.HASHDIFF
-            WHERE latest_records.HASHDIFF IS NULL
+                ON latest_records.CUSTOMER_PK = stage.CUSTOMER_PK
```

Comment:
Tab this correctly, align with other SQL above.

Reply:
This is a change in the sat section of the documentation, not mas. The example sql here for sat in fact has also not been updated to reflect the current version of sat.sql, and sat.sql itself has a layout issue in this section. So both the documentation and the feature code for sat need minor corrections. Perhaps as this is a MAS PR this should be allowed through as is and the corrections made as bug fixes through future PRs?